code/style.css

```diff
@@ -1,5 +1,55 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap');
 
-body {
-  background-color: lightblue;
+menu{
+    color: rgb(85,87,83);
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
 }
+header{
+    background-color: rgb(46,139,87);
+    padding: 20px;
+    width: 1200px;
+    border: 5px,rgb(85,87,83);
+    margin: 10px;
+}
```

Comment:
When I try to open your site on a bigger screen, your nav menu succeeds at taking all the available space, but your header doesn't. Removing the predetermined width here solved it. 📏🙂
![review1](https://user-images.githubusercontent.com/70166033/137203380-5971e28b-8165-4ea9-82c9-ee1cbdc8b420.jpg)

![review2](https://user-images.githubusercontent.com/70166033/137203420-804ca4de-94b6-4049-8baa-5bd5b5abdbbf.jpg)